app/waf/src/models/as3.model.ts

```diff
@@ -41,4 +41,17 @@ export class AS3DeployRequest {
       });
     }
   }
+
+  static generateDeployBody(declaration: Declaration): object {
```

Comment:
Is it duplicated with the constructor?

Reply:
If we don't use AS3, the AS3* staff will be obsolete. 
Also, I don't think generating a deployment body, `adc application` must be provided.
What's more, I just think that this part can be simplified to a function.

Reply:
We still need the code to create AS3 request, but we just post the AS3 request via proxy.

Reply:
Reused the AS3DeployRequest constructor.